src/providers/persistence/persistence.ts

```diff
@@ -100,7 +100,7 @@ export class PersistenceProvider {
   public load() {
     this.storage = this.platform.isCordova
       ? new FileStorage(this.file, this.logger)
-      : new LocalStorage(this.platform, this.logger);
+      : new LocalStorage(this.logger);
```

Comment:
Can we test that Electron support >1MB data per app?

Reply:
I did a manual test running a script and it supports up to 5MB.